tests/cleaners/test_remove.py

```diff
@@ -823,3 +823,13 @@ def test_remove_arabic_letter_dots_with_edge_case(input: str, expected: str):
 def test_remove_arabic_letter_dots_general(input: str, expected: str):
 
     assert remove_arabic_letter_dots(input) == expected
+
+
+def test_remove_list_input(simple_text_input: str):
+    list_ = [r"ال", r"the"]
```

Comment:
I think It would be better if we added one regular expression to the list as well.